### PR TITLE
fix issues in kairship-faq doc

### LINKED ISSUE #5436: 【多云编排】-常见问题，几个优化的点
https://docs.daocloud.io/kairship/intro/faq/

多云编排-常见问题，几个优化的点

问题4 - 能否给一下在哪里查看的说明

问题7 - TP是什么阶段，缩写有点看不懂

image

问题11 - 能不能不使用口语化的说明？

image

问题12 - 在哪里进行配置，截图中能否把命令的详细设置给出？

image

### PR DESCRIPTION
**Pull request by ModetaNiu**
Fixed #5436, Require review @wlp1153468871

### CODE REVIEW

**Comment by ModetaNiu** [comment]
Preview: 
https://deploy-preview-5468--daocloud-docs.netlify.app/kairship/intro/faq

File: docs/zh/docs/kairship/intro/faq.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ hide:
 
 4. 分发到多个集群的工作负载，是否可以在一个视图呈现监控信息？
 
-    支持在统一的视图来查看多云应用，可以监控他们部署到了什么集群，对应的 Service、传播策略等等。
+    支持在统一的视图来查看[多云工作负载](../workload/deployment.md)，可以监控他们部署到了什么集群，对应的 Service、部署策略等等。
```
Comment (windsonsea) [inline docs/zh/docs/kairship/intro/faq.md:22]:
```suggestion
    支持在统一的 UI 上增删改查[多云工作负载](../workload/deployment.md)，了解这些负载被部署到了什么集群，对应的 Service、部署策略等等。
```

**Review by windsonsea — COMMENTED** [review]
nit

**Review by windsonsea — APPROVED** [review]

**Pull request merged**